components/post/default.htm

```diff
@@ -18,11 +18,13 @@
 
 <p class="info">
     {% if post.categories.count %}
-        {% set categoryLinks = post.categories | map(c => "<a href=\"#{c.url}\">#{c.name}</a>") | join(', ') %}
-
+        {% set categoryLinks = [] %}
+        {% for category in post.categories %}
+            {% set categoryLinks = categoryLinks|merge(["<a href=\"#{category.url}\">#{category.name}</a>"]) %}
```

Comment:
Can you show me what the output is? I don't know why there is a `#` sign in there.

Reply:
I'm going to have to figure out a way to test it. I haven't actually seen this page in action, it was just causing me issues on my php 7.1 system.

Reply:
@LukeTowers I believe `#` is used to denote an inline variable in a string in Twig.